src/sentry/search/utils.py

```diff
@@ -595,19 +609,43 @@ def split_query_into_tokens(query):
                 if quote_enclosed:
                     quote_type = char
         if quote_enclosed and char == "\\" and next_char == quote_type:
-            token += next_char
+            token += next_char  # type: ignore[operator]
```

Comment:
`token` and `next_char` are both defined in this function. This shouldn't be an error and if it is, then this function has a bug.